pnpm-lock.yaml

```diff
@@ -20152,7 +20097,7 @@ packages:
       assert-never: 1.2.1
       globby: 13.2.2
       prettier: 3.0.3
-      typescript: 5.3.2
+      typescript: 5.3.3
```

Comment:
Can you revert his change? I don't think this is required?

Reply:
Sure! It came with rebase and re-built, is that any risk to upgrade the package?